styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/03-card-row-ec-five-up-left.twig

```diff
@@ -1,9 +1,19 @@
 <div class="layout layout_five-up-left">
-  <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
+  {% if cardType == 'landscape' %}
+  <div class="layout_item">{% include 'molecules-card-image-landscape' with {'cardRowContent': cardRowContent.cardsLandscape.first} %}</div>
```

Comment:
Addressed in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).